framework/decode/file_processor.cpp

```diff
@@ -1290,7 +1290,7 @@ bool FileProcessor::ProcessAnnotation(const format::BlockHeader& block_header, f
         {
             std::string label;
             std::string data;
-            auto        total_length = label_length + data_length;
+            size_t      total_length = size_t(label_length + data_length);
```

Comment:
Please make this a C++ cast

Reply:
Will do. I'd mislearned that these were construction, not casting, so thanks!

All this casting is to supress warnings as errors in 32 bit windows builds. This reminds me, our 32 bit builds are in trouble if any 64 bit fields in our capture format come in with values over 4GB. I don't know that we try to handle that anywhere, even to log and fail gracefully.

Reply:
Applied the macro and changed the casts.